src/main/java/edu/greenblitz/robotName/subsystems/Lifter/ILifter.java

```diff
@@ -0,0 +1,15 @@
+package edu.greenblitz.robotName.subsystems.Lifter;
+
+import com.revrobotics.CANSparkMax;
+
+public interface ILifter {
+    public void setPower(double power);
+    public void setVoltage(double voltage);
+    public void resetEncoderTo(double position);
+    public boolean isMotorInPosition(double position);
```

Comment:
in the Interface put only void functions. you have the inputs class for the others